catalog/networking/dns/recordset/Kptfile

```diff
@@ -1,71 +1,17 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
 kind: Kptfile
 metadata:
   name: dns-record-set
-openAPI:
-  definitions:
-    io.k8s.cli.setters.namespace:
-      description: Namespace configured for managing resources defined in this package
-      x-k8s-cli:
-        setter:
-          name: namespace
-          value: networking
-    io.k8s.cli.setters.project-id:
-      description: Project ID for record-set
-      x-k8s-cli:
-        setter:
-          name: project-id
-          value: project-id
-    io.k8s.cli.setters.record-set-name:
-      description: Record Set name
-      x-k8s-cli:
-        setter:
-          name: record-set-name
-          value: record-set-example
-    io.k8s.cli.setters.name:
-      description: The name of this record-set
-      x-k8s-cli:
-        setter:
-          name: name
-          value: ""
-    io.k8s.cli.setters.domain:
-      description: The DNS domain of this record-set
-      x-k8s-cli:
-        setter:
-          name: domain
-          value: example.com.
-    io.k8s.cli.setters.type:
-      description: The DNS record set type
-      x-k8s-cli:
-        setter:
-          name: type
-          value: A
-    io.k8s.cli.setters.ttl:
-      description: The time-to-live of this record set (seconds)
-      x-k8s-cli:
-        setter:
-          name: ttl
-          value: "300"
-    io.k8s.cli.setters.managed-zone-name:
-      description: Managed Zone to bind to
-      x-k8s-cli:
-        setter:
-          name: managed-zone-name
-          value: foo
-    io.k8s.cli.setters.records:
-      type: array
-      x-k8s-cli:
-        setter:
-          name: records
-          listValues:
-          - "0.0.0.0"
-    io.k8s.cli.substitutions.rr-domain:
-      x-k8s-cli:
-        substitution:
-          name: rr-domain
-          pattern: ${name}${domain}
-          values:
-          - marker: ${name}
-            ref: '#/definitions/io.k8s.cli.setters.name'
-          - marker: ${domain}
-            ref: '#/definitions/io.k8s.cli.setters.domain'
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/apply-setters:v0.1
+      configMap:
+        domain: example.com.
+        managed-zone-name: foo
+        name: ""
+        namespace: networking
+        project-id: project-id
+        record-set-name: record-set-example
+        records: '[0.0.0.0]'
```

Comment:
Not sure if this is the right format for list setters based on docs https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/apply-setters#:~:text=declare%20the%20desired%20array%20values%2C%20wrapped%20into%20string.

Reply:
This seems to work and it's what was auto-generated. @phanimarupaka Can you clarify? It seems the docs and fix function are inconsistent.

Reply:
These are just different formats, this is flow-style and the one in the docs is folded style. Both are acceptable yaml formats. But I agree that it would cause confusion. Here is the issue for tracking https://github.com/GoogleContainerTools/kpt/issues/2291